appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -594,6 +604,7 @@ public void onProjectLoaded(Project glass) {
         assetManager = AssetManager.getInstance();
       }
       assetManager.loadAssets(project.getProjectId());
+      bindAssetListBox.getAssetList().refreshAssetList(project.getProjectId());
```

Comment:
In master this is called from YaProjectEditor onHide and onShow, why is this now needed here?

Reply:
> In master this is called from YaProjectEditor onHide and onShow, why is this now needed here?

I strongly suspect this can be reverted, but we'll have to try it and make sure.

Reply:
It looks like the asset list refresh is still in the YaProjectEditor methods, so I don't think this needs to be here. Removing.